metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -319,6 +319,7 @@ object Messages {
         params: ShowMessageRequestParams,
         details: Map[String, BspConnectionDetails]
     )
+    // TODO This is currently only being used in tests. This can probably be removed
```

Comment:
I'll need to do a bit more work here (in a follow-up pr if that's ok) since this is only being used in tests. I need to remove them, but I also want to replace them with stuff that is actually testing thing we want. I plan to expand the `bsp-switch` suite and also add in a `BspConnector` suite.